test/aws/default/verify/controls/aws_route_tables.rb

```diff
@@ -0,0 +1,28 @@
+fixtures = {}
+[
+  'routetable_rtb_route_table_id',
```

Comment:
These seem like rather long, confused names for fixtures.  The usual pattern is `<resource_type>_<fixture_name>_<property_name>` So you could shorten it to `route_table_1_id`, `route_table_2_vpc_id`, etc.